src/containers/familyDetails/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Alert, Spin } from 'antd';
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import protectedApiClient from '../../api/protectedApiClient';
+import { ChungusContentContainer } from '../../components';
+import ContactInfoSummary from '../../components/ContactInfoSummary';
+import {
+  AsyncRequest,
+  AsyncRequestCompleted,
+  AsyncRequestFailed,
+  asyncRequestIsComplete,
+  asyncRequestIsFailed,
+  asyncRequestIsLoading,
+  asyncRequestIsNotStarted,
+  AsyncRequestLoading,
+  AsyncRequestNotStarted,
+} from '../../utils/asyncRequest';
+import { ContactInfo } from '../setContacts/ducks/types';
+
+const FamilyDetails: React.FC = () => {
+  const dispatch = useDispatch();
+  const id = Number(useParams<{ id: string }>().id);
+  const [contacts, setContacts] = useState<AsyncRequest<ContactInfo, any>>(
+    AsyncRequestNotStarted(),
+  );
+
+  useEffect(() => {
+    if (asyncRequestIsNotStarted(contacts)) {
```

Comment:
Just a note about this, I haven't worked with this code base at all so take it with a grain of salt. In HATS we try to minimize component state, and move as much state as possible to Redux. I'm not familiar with the design patterns for LLB, but in general I think the "when in doubt, move state up to Redux" is a good philosophy for React.

Reply:
On this page, we're just fetching data and displaying it, without any update, create, or delete functionality. The data isn't needed globally, although in the future it will be so that families can select who specifically is coming to an event.